exec-unittests

```diff
@@ -138,26 +144,29 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
         fi
     done
 
-    printf -- "\n\n\nRunning %s python3-based unit tests via pytest\n\n" "${major_list// /,}"
-    _pbench_sources=$(python3 -c 'import inspect, pathlib, pbench; print(pathlib.Path(inspect.getsourcefile(pbench)).parent)')
-    _PBENCH_COV_DIR="${_toxenvdir}/cov" _time pytest \
-        ${pytest_jobs_arg} \
-        --basetemp="${_toxenvdir}/tmp" \
-        --cov=${_pbench_sources} \
-        --cov-report ${_cov_report} \
-        ${posargs} \
-        --pyargs ${_pytest_majors}
-    rc=${?}
-    if [[ ${rc} -ne 0 ]]; then
-        printf -- "\n%s pytest command failed with '%s'\n\n" "${_major^}" "${rc}"
-    else
-        printf -- "\n%s pytest command succeeded\n\n" "${_major^}"
-    fi
+    if [[ -n "${_pytest_majors}" ]]; then
+        _pytest_majors="pbench.test.unit.common${_pytest_majors}"
```

Comment:
I can't figure it out:  when will `_pytest_majors` ever be empty at line 147?  (I'm inclined to suggest that you restore the change to line 130 and omit lines 147-148.)